### PR TITLE
Local plugin to fix translated mdx images urls

### PR DESCRIPTION
**Pull request by pettinarip**
## Description
The following proposal tries to solve having different images urls between source and translated md files. With this, we won't need to adjust the images urls in translated files (we wil keep the same as in the source file).

## Solution
By using a local gatsby plugin we can intercept the nodes created by `gatsby-plugin-mdx` and adjust the images urls. We can think about it as a "translator" for urls.
- If we are in a source file => the plugin will ignore it
- If we are in a translated file => the plugin will modify the url by pointing them to the source file folder (what we were doing manually)

Check working example https://ethereumorgwebsitedev01-mdximagespaths.gtsb.io/ro/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/

## Related Issue
#2585 

## TODO
- [x] Change variable names that contains `path` with `url`
- [x] Add more docs

### CODE REVIEW

File: src/content/translations/ro/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ source: Mediu
 sourceUrl: https://medium.com/alchemy-api/getting-started-with-ethereum-development-using-alchemy-c3d6a45c567f
 ---
 
-![Logouri Ethereum și Alchemy](../../../../../developers/tutorials/getting-started-with-ethereum-development-using-alchemy/ethereum-alchemy.png)
+![Logouri Ethereum și Alchemy](./ethereum-alchemy.png)
```
Comment (pettinarip) [inline src/content/translations/ro/developers/tutorials/getting-started-with-ethereum-development-using-alchemy/index.md:22]: The plugin will basically translate `./ethereum-alchemy.png` into `../../../../../developers/tutorials/getting-started-with-ethereum-development-using-alchemy/ethereum-alchemy.png`

**Review by wackerow — APPROVED** [review]
@pettinarip This is great! Nice and clean.

This appears to be backward compatible, so the existing images are not breaking, which is great. I think we bring this in and separately we can make an issue to clean up all the existing images.

Last think would just be to double check w/ @lukassim if we need to re-upload any of these to Crowdin. The files should have the short relative paths by default, but just want to double check we didn't manually overwrite any of those paths. In either case, we should be good here! 👍

**Comment by pettinarip** [comment]
> I think we bring this in and separately we can make an issue to clean up all the existing images.

@wackerow this approach allows us to avoid touching the current files. We can keep everything as it is.

My idea was that starting from now, the files will progressively update themselves to the short relative path. We shouldn't necessarily do any new PR to manually update the files.

Maybe is something we can discuss as a team:
- Do we prefer to update all the translated files at once to start using the same images urls that the source files have?
- Or do we prefer to keep everything as it is and while we upload newly translated files the urls are going to be updating themselves?

**Review by corwintines — APPROVED** [review]
Looks great, thanks for linking out to gatsby stuff for context :)

**Comment by minimalsm** [comment]
> The files should have the short relative paths by default, but just want to double check we didn't manually overwrite any of those paths. In either case, we should be good here! 👍

@wackerow I've manually overwritten all of these paths (so when you download from Crowdin you get the previously correct path (e.g. `../../../../../developers/tutorials/getting-started-with-ethereum-development-using-alchemy/ethereum-alchemy.png`)

But, this is a relatively easy fix that requires 0 review from Acolad :-) the main concern would be if there are other changes to the source content that might set back progress in the markdown files with images.

**Comment by corwintines** [comment]
Merging based on conversation in standup today

**Pull request merged**